src/sections/Community/Handbook/writing-program/content-form.style.js

```diff
@@ -29,7 +29,12 @@ const ContentFormWrapper = styled.section`
         font-size: .9rem;
         margin: 0 1rem 1rem;
         padding: 1rem;
-        
+        overflow: auto;
+        height: 600px;
+        white-space: nowrap;
```

Comment:
@vishnus17, I scratch my head at this change. It doesn't "feel" right to me. 🤔

Reply:
That was added just to make sure that text doesn't overflow and stays in the line in which it is supposed to stay as mentioned in the code.

Reply:
Looking at it I think it should be removed as the slack link is not entirely visible because of it. I thought the contents were coded such that it preserves the width.

Reply:
Thanks for the catch @leecalcote. It's not required. I tested the responsiveness of the form as well now.
I'll remove it and make update the PR right away! 🙌